help/tquery.md

```diff
@@ -0,0 +1,52 @@
+# Tquery
+
+### Typy danych i operatory filtrów
+
+| typ          | null(5) | = | in   | cmp | like | select | sort |
```

Comment:
Dodaj regexp. Chyba tylko dla string i text. Pewnie dla dat i intów też by działało ale raczej bym uznał że to nie ma sensu. Regexp powinien działać tak jak ten == który proponujesz, bez żadnego automatycznego czegokolwiek.